### ISSUE TITLE
Add support for an operator board

### ISSUE THREAD

**Issue by LennyAbbas** [SaintsRobotics/RapidReact2022#108]

**Comment by bharadwajduggaraju** [comment]
I don't think that any of our operator have expressed interest in using an operator board, I think that one of the operators actually didn't want to use it, so this feature should have like a long term project tag, since their seems to be a lot of bugs that might have to do with the hardware of the operator board itself.

**Comment by bharadwajduggaraju** [comment]
I know there is a PR for this, is it working? If so then my entire message is invalid I didn't see that until now.
